src/EventListener/AngularHtmlListener.php

```diff
@@ -79,12 +81,16 @@ function ($match) use ($file, $reader) {
      */
     private function getCompiledAssetFor(string $linked_file, File $owning_file, ReaderInterface $reader): string
     {
-        $file_path = $linked_file;
-
-        if ($file_path[0] === '.' && $owning_file->dir !== '.' && !empty($owning_file->dir)) {
-            $file_path = $owning_file->dir . '/' . $file_path;
+        if (substr($linked_file, 1, 0) === '/') {
```

Comment:
```bash
php -r "var_dump(substr('ABC', 1, 0));"
```
```
string(0) ""
```
Based on that, this if can not become true. There should probably be a unit-test for this case.